.github/workflows/test.yml

```diff
@@ -167,12 +167,12 @@ jobs:
             django: 'Django>=4.2,<4.3'
             experimental: false
             mysql: 'mariadb:10.5'
-            healthcmd: 'mariadb-admin ping'
+            healthcmd: 'healthcheck.sh --connect --innodb_initialized'
```

Comment:
Hmmm at this point I think we should use something else rather than abusing the matrix config, e.g. an environment variable with different values based on `matrix.mysql`. Otherwise this shows up in the checks summary, making it unnecessarily verbose:

<img width="667" alt="image" src="https://github.com/user-attachments/assets/a8fd8925-5d44-496c-8c0a-664bb3fb52f7">